### PR TITLE
boulder: Bring data/ up to parity with boulder-d

### PR DESCRIPTION
**Pull request by ermo**
Note that a new x86_64-v3x.yml target definition was added in preparation for the point where we will begin building the scaffolding to use it.

### CODE REVIEW

File: boulder/data/default-x86_64.conf [hunk 1]
```diff
@@ -0,0 +1,20 @@
+- default-x86_64:
+    collections:
```
Comment (tarkah) [inline boulder/data/default-x86_64.conf]: We use `yaml` file extension for our config files now since there... yaml :)

Comment (tarkah) [reply boulder/data/default-x86_64.conf]: Also, we don't use the weird array / map nesting, so the `-` should be dropped from the map keys

Comment (tarkah) [reply boulder/data/default-x86_64.conf]: Shit.. we already have `boulder/data/profile.d/default-x86_64.yaml` :)

Comment (ermo) [reply boulder/data/default-x86_64.conf]: the `.yml` vs. `.yaml` thing is getting on me nerves (I personally prefer `.yaml`).

Comment (livingsilver94) [reply boulder/data/default-x86_64.conf]: The standard technically imposes `.yaml`. We should just follow that extension and call it a day, it's a no-brainer.

Comment (ermo) [reply boulder/data/default-x86_64.conf]: ... We should probably use `.yaml` project wide as well then (assuming this doesn't make @ikeycode blow a gasket!)

**Review by tarkah — APPROVED** [review]

**Pull request merged**